src/store.js

```diff
@@ -147,11 +147,17 @@ class AsyncStorageStore {
    */
   async clearItems(onlyWalletKeys = false) {
     const keys = await AsyncStorage.getAllKeys() || [];
-    this.hathorMemoryStorage = {};
```

Comment:
The actual source of truth is the `AsyncStorage` so cleaning this will not erase the storage, unless we try to read before repopulating the `hathorMemoryStorage` which was happening, to fix this we only delete the required keys